nautobot_golden_config/models.py

```diff
@@ -486,21 +495,33 @@ def get_absolute_url(self):  # pylint: disable=no-self-use
 
     def __str__(self):
         """Return a simple string if model is called."""
-        return "Golden Config Settings"
+        return "Configuration Object"
```

Comment:
Configuration Object loses the context of the fact that this is part of Golden Config, perhaps the old one can use an additional descriptor, but given the two I prefer the previous one.

Reply:
See previous comment to Jeff's review.

Old pattern has the stringified instance of an object as the same as the stringified model. This causes duplicated verbiage in views. Edit view will have a heading of `Golden config settings Golden Config Settings` and it is <model str> <instance str>